bin/morgue.js

```diff
@@ -4755,7 +4755,10 @@ async function coronerSimilarity(argv, config) {
   }
   
   const coroner = coronerClientArgv(config, argv);
-  const similarityEndpoint = `${coroner.endpoint}${similarityService.endpoint}`;
+  const isAbsoluteEndpoint = similarityService.endpoint.startsWith('http');
+  const similarityEndpoint = isAbsoluteEndpoint ?
+      similarityService.endpoint :
+      `${coroner.endpoint}${similarityService.endpoint}`;
```

Comment:
```suggestion
  const similarityEndpoint = similarityService.endpoint.startsWith('http') 
     ? similarityService.endpoint 
      : `${coroner.endpoint}${similarityService.endpoint}`;
```

One more suggestion - what do you think about this code? One less line and it will be alright with our formatting rules

Reply:
sure, but clang-format prefers this one:
```
  const similarityEndpoint = similarityService.endpoint.startsWith('http') ?
      similarityService.endpoint :
      `${coroner.endpoint}${similarityService.endpoint}`;
```

do you want to move `?` to the next line like so?
```
  const similarityEndpoint = similarityService.endpoint.startsWith('http') 
      ? similarityService.endpoint 
      : `${coroner.endpoint}${similarityService.endpoint}`;
```

Reply:
I went with this one:
```
const similarityEndpoint = similarityService.endpoint.startsWith('http') ?
      similarityService.endpoint :
      `${coroner.endpoint}${similarityService.endpoint}`;
```